src/realtimefeeds/bittrex.ts

```diff
@@ -0,0 +1,130 @@
+import got from 'got'
+import { wait } from '../handy'
+import { Filter } from '../types'
+import { MultiConnectionRealTimeFeedBase, RealTimeFeedBase } from './realtimefeed'
+
+abstract class BittrexRealTimeFeedBase extends MultiConnectionRealTimeFeedBase {
+  protected abstract wssURL: string
+  protected abstract httpURL: string
+  protected abstract suffixes: { [key: string]: number }
+
+  protected *_getRealTimeFeeds(exchange: string, filters: Filter<string>[], timeoutIntervalMS?: number, onError?: (error: Error) => void) {
+    const wsFilters = filters.filter((f) => f.channel !== 'depthSnapshot')
```

Comment:
Won't this always prevent the snapshot from being fetched?

Reply:
Good catch, was testing and forget to remove it ^^